assets/src/web-stories-block/block/block.json

```diff
@@ -1,6 +1,18 @@
 {
+  "apiVersion": 2,
   "name": "web-stories/embed",
+  "title": "Web Stories",
+  "description": "Embed Web Stories.",
   "category": "embed",
+  "keywords": [
+    "embed",
+    "web-stories",
+    "web stories",
+    "web-stories",
```

Comment:
^ There's a duplicate. Also, would `web-stories` vs `web stories` give a different result? Feels like these three could just be `web stories`.

Reply:
Fixed.

Reply:
@spacedmonkey It still has `"web-stories"` and `"web stories"` separately -- isn't this the same thing as a keyword? Just `web stories` should work for both cases, no?

Reply:
I thought that was in the orignal. It wasn't removed now.